core/assets/bundles/bundle.properties

```diff
@@ -1639,4 +1639,4 @@ lenum.mine = Mine at a position.
 lenum.build = Build a structure.
 lenum.getblock = Fetch a building and type at coordinates.\nUnit must be in range of position.\nSolid non-buildings will have the type [accent]@solid[].
 lenum.within = Check if unit is near a position.
-lenum.boost = Start/stop boosting.
\ No newline at end of file
+lenum.boost = Start/stop boosting.
```

Comment:
You inserted an unnecessary line break at the end of this line.

Reply:
This was done in the web editor, GitHub did that automatically. Any ideas on how to fix?

Reply:
Don't use the web editor to edit files.

Reply:
Sounds good.

Reply:
#teamlinebreaks ❤️